### DISCUSSION TITLE
Q-Stepper Line modifications

### DISCUSSION THREAD

**Discussion by mtn090899** [quasarframework/quasar#13577] [General - Components / Directives / etc]
Hi,

So I'm utilizing the Q-Stepper component, but I want to make some styling changes that don't seem to be present within the Quasar documentation, and I wanted to know if it was possible/how to make these changes.  This is what I have with Q-Stepper currently.

![image](https://user-images.githubusercontent.com/78068147/171209442-30a9e3f3-b39d-4d35-a47a-c5018627c7f7.png)

I want to make the lines in between each step thicker, touching each step, and able to change color upon being passed, although I don't know where to start.

Thank you!

**Comment by MilosPaunovic** [comment]
Start by inspecting element in the browser console.

You can use this as a starting point:
[https://codepen.io/milospaunovic/pen/OJQvGyJ](https://codepen.io/milospaunovic/pen/OJQvGyJ)

**Reply by mtn090899** [reply]
Thanks! I tried that, but I guess I had trouble selecting the element in browser console. Playing with that brought up another issue. I need to make the lines able to touch the steps, however it seems like there's always something making a gap in between and I can't figure out what.

![image](https://user-images.githubusercontent.com/78068147/171231510-af2f8c36-2661-4575-b9b9-6285e2356e45.png)
